### PR TITLE
Autopilot networking tutorial manifest changes

### PR DESCRIPTION
**Pull request by melissakendall**
Fixes https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/issues/571

Summary of changes:
- Remove any references to "simple" or "basic" as those words are not advised as per Google developer style guide
- Removed "bank of anthos" as the tutorial that uses these manifests does not modify the standard Bank of Anthos app, and cloning 2 repos was unnecessary. Best to keep everything in one repo, and make it simpler.
- Added Terraform snippets. NOTE: I did not create these, and have limited ability to test them. I am relying on SWEs to identify if there are any errors.
- Changed object naming to make it easier to track objects (instead of everything being named "bank-of-anthos")

I'd love to squash my commits, but I appear unable to do that through GitHub UI. I think I can do it when I merge the PR though.

### CODE REVIEW

File: autopilot/networking-tutorial/dns.tf [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+variable "name" {
+  type        = string
+  description = "Name of resources"
+  default     = "networking-tutorial"
+}
+
+data "google_client_config" "current" {}
+
+resource "google_compute_global_address" "default" {
+  name = var.name
```
Comment (melissakendall) [inline autopilot/networking-tutorial/dns.tf:35]: Does this mean that the global IP address that gets created will have the standard name "networking-tutorial"? If so I should change this to match the IP address referenced in kubernetes-manifests.yaml.

**Comment by melissakendall** [comment]
Incorporated revisions from a GKE subject matter expert who tested the Terraform snippets.

**Comment by bourgeoisor** [comment]
@melissakendall `networking-tutorial` seems a bit vague as the directory name. I'm "okay" with it but I'm worried that eventually we may have samples for another "networking tutorial" and naming may become confusing. Is there an alternative we could use?

**Comment by melissakendall** [comment]
Fair enough. "basic networking tutorial"?

I just dont want to add "bank of anthos" because the goal of the tutorial is to resolve a concern from customers that setting up an AP cluster is easy but adding all the networking requirements is difficult. To me, that means that adding BoA would add too much complexity (and the customer would have to checkout an additional repo)

**Comment by melissakendall** [comment]
> @melissakendall `networking-tutorial` seems a bit vague as the directory name. I'm "okay" with it but I'm worried that eventually we may have samples for another "networking tutorial" and naming may become confusing. Is there an alternative we could use?

Fair enough. "basic networking tutorial"?

I just dont want to add "bank of anthos" because the goal of the tutorial is to resolve a concern from customers that setting up an AP cluster is easy but adding all the networking requirements is difficult. To me, that means that adding BoA would add too much complexity (and the customer would have to checkout an additional repo)

**Comment by bourgeoisor** [comment]
What's the name of the tutorial? Maybe we can play on that!
Agreed with BoA being complex for more intro tutorials that don't require multiple services, keeping it simple works wonders

**Comment by melissakendall** [comment]
> What's the name of the tutorial? Maybe we can play on that! Agreed with BoA being complex for more intro tutorials that don't require multiple services, keeping it simple works wonders

Currently: 
Configure networking for a basic production cluster

But I am not sold on "basic" or "production"

**Review by bourgeoisor — APPROVED** [review]
Still unsure about the directory name `networking-tutorial` but I don't feel super strongly about it. LGTM! Let me know when / if it's time to merge this.

**Comment by melissakendall** [comment]
Ready to submit at any time!

**Pull request merged**